rmgpy/statmech/torsion.pyx

```diff
@@ -530,7 +530,7 @@ cdef class HinderedRotor(Torsion):
         # because it is the number of terms of either the cosine or sine fit
 
         maxterms = np.floor(len(angle) / 3.0)
-        while negative_barrier and numterms < maxterms:
+        while negative_barrier and numterms <= maxterms:
```

Comment:
would you like to squash this commit into the previous one?

Reply:
Sure thing.